etc/config/c++.amazon.properties

```
@@ -3858,6 +3858,12 @@ libs.belleviews.description=A library of C++ views that just works for all basic
 libs.belleviews.versions.trunk.version=trunk
 libs.belleviews.versions.trunk.path=/opt/compiler-explorer/libs/belleviews/trunk/sources
 
+libs.beman_iterator_interface.name=beman_iterator_interface
+libs.beman_iterator_interface.versions=trunk
+libs.beman_iterator_interface.url=https://github.com/beman-project/iterator_interface
+libs.beman_iterator_interface.versions.trunk.version=trunk
+libs.beman_iterator_interface.versions.trunk.path=/opt/compiler-explorer/libs/beman_iterator_interface/main/include
+
 libs.beman_optional26.name=beman_optional26
```

```suggestion
libs.beman_optional26.name=beman.optional26
```

Please apply same updates for beman.optional26 (https://github.com/beman-project/Optional26?tab=readme-ov-file#bemanoptional26-c26-extensions-for-stdoptional)